### DISCUSSION TITLE
how  I can receive data with post  method

### DISCUSSION THREAD

**Discussion by mrdsn3** [esp8266/Arduino#8003] [Q&A]
I search every where  but cant find my answer 
I want to receive  data  with post method for ath  but dont know how

**Comment by d-a-v** [comment]
client: https://github.com/esp8266/Arduino/tree/master/libraries/ESP8266HTTPClient/examples/PostHttpClient
server: https://github.com/esp8266/Arduino/tree/master/libraries/ESP8266WebServer/examples/PostServer
